README.md

```diff
@@ -2,20 +2,24 @@
 
 Multi Cluster scheduler, designed to efficiently manage and schedule AI/ML, Bigdata workloads across multiple Kubernetes clusters.
 
-# volcano-global
-
-volcano-global is a multiple kubernetes clusters scheduler, designed to efficiently manage and schedule AI/ML,
+Volcano-global is a multiple kubernetes clusters scheduler, designed to efficiently manage and schedule AI/ML,
 Bigdata workloads across multiple kubernetes clusters,
 which cleverly combines the capabilities of [Karmada](https://karmada.io/) and [Volcano](https://volcano.sh/en/)
 
 This project originated from:
 - [LFX Mentorship CNCF - Volcano: Volcano supports multi-cluster AI workloads scheduling](https://mentorship.lfx.linuxfoundation.org/project/132a4971-6969-4ca6-a695-783ece3ac768)
 - [OSPP 2024 Volcano supports queue capacity management capabilities in multi-cluster AI workload scheduling](https://summer-ospp.ac.cn/org/prodetail/243ba0505?list=org&navpage=org)
 
+> This is an alpha version and code is subject to change.
+> 
 # Quick start
 
 You can read [deploy guide](docs/deploy/README.md).
 
 # How does it run?
 
-![volcano global design](docs/imgs/volcano_global_design.png)
+![volcano global design](docs/imgs/volcano_global_design.svg)
+
+# Release Guide
+
+The release cadence of the `volcano-global` is not synchronized with that of [Volcano](https://github.com/volcano-sh/volcano). This is because the `volcano-global` is a sub-repository under volcano-sh, and its codes and features are developed independently of the main repo. We will release new versions as needed.
```

Comment:
It's not explict to use `synchronized`.  What you intend to say is like below?

volcano-global is a seperated sub-project, having it's own release. The project plays well with volcano repo. However the release are not coupled.

Reply:
done.